code/modules/security/crimes.dm

```diff
@@ -0,0 +1,274 @@
+#define MAX_TIMER 10 HOURS //Permabrig.
+#define PRESET_SHORT 5 MINUTES
+#define PRESET_MEDIUM 10 MINUTES
+#define PRESET_LONG 15 MINUTES
+
+#define CRIME_MINOR "Minor"
+#define CRIME_MISDEMEANOUR "Misdemeanour"
+#define CRIME_MAJOR "Major"
+#define CRIME_CAPITAL "Capital"
+
+/* This is overwritten by the config (space_law.json as of writting this), this only exists as a backup in case the game fails to load said file.
+Do not modify this unless you know what you're doing. */
```

Comment:
(Doesn't need changes, just a comment)
I mean technically speaking, you don't really need to know what you are doing since these are type checked so anything that compiles should in theory  be fine.